RELEASE_PROCESS.md

```diff
@@ -1,36 +1,131 @@
+<!-- omit from toc -->
+
 # Releasing a new version of Spectrum Web Components
 
-Users with permissions in the `@spectrum-web-components` organization on NPM can follow the following steps to create and publish a new version.
+Users with permissions in the `@spectrum-web-components` organization on NPM can follow these steps to create and publish a new version.
+
+-   [Prerequisites](#prerequisites)
+    -   [Main successfully builds](#main-successfully-builds)
+    -   [The correct version of Node is installed](#the-correct-version-of-node-is-installed)
+        -   [Using Node Version Manager](#using-node-version-manager)
+        -   [Manually checking](#manually-checking)
+        -   [Troubleshooting](#troubleshooting)
+    -   [Github Token is set up](#github-token-is-set-up)
+        -   [Generate a Github token](#generate-a-github-token)
+    -   [Logged in to NPM](#logged-in-to-npm)
+    -   [NPM 2FA authenticator app](#npm-2fa-authenticator-app)
+-   [Releasing to NPM — the good stuff](#releasing-to-npm--the-good-stuff)
+    -   [Troubleshooting](#troubleshooting-1)
+-   [Publishing the documentation site manually](#publishing-the-documentation-site-manually)
+    -   [From GitHub](#from-github)
+    -   [From the terminal](#from-the-terminal)
+    -   [References](#references)
+
+## Prerequisites
+
+### Main successfully builds
+
+1. Merge all pull requests to be included in the release, and wait for the `main` branch to show that it has completed the required Circle CI jobs.
+    1. Check [Circle Ci build for `main`](https://app.circleci.com/pipelines/github/adobe/spectrum-web-components?branch=main) shows a `success` status
+        1. If it failed, click `rerun` dropdown and select `rerun from failed`.
+        2. If it continues to fail, investigate further until you can successfully get the `main` branch building.
```

Comment:
```suggestion
Merge all pull requests to be included in the release, and wait for the `main` branch to show that it has completed the required Circle CI jobs.

Check [Circle Ci build for `main`](https://app.circleci.com/pipelines/github/adobe/spectrum-web-components?branch=main) shows a `success` status.
    1. If it failed, click `rerun` dropdown and select `rerun from failed`.
    2. If it continues to fail, investigate further until you can successfully get the `main` branch building.
```

Small nit since there wasn't more than 1 item in the list, I thought it might be more readable without the indent.